core/trino-main/src/test/java/io/trino/sql/planner/optimizations/TestRemoveUnsupportedDynamicFilters.java

```diff
@@ -76,9 +75,7 @@
 import static io.trino.sql.planner.assertions.PlanMatchPattern.values;
 import static io.trino.sql.planner.plan.JoinType.INNER;
 import static io.trino.type.UnknownType.UNKNOWN;
-import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
 
-@TestInstance(PER_CLASS)
```

Comment:
same here, `PlanTester` is not lightweight - add `Execution(CONCURRENT)` instead.

Reply:
same answer -- BasePlanTest is `@TestInstance(PER_CLASS) @Execution(CONCURRENT)`
(just like ATQF)